streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/impl/KubernetesNativeSessionSubmit.scala

```diff
@@ -212,4 +212,9 @@ object KubernetesNativeSessionSubmit extends KubernetesNativeSubmitTrait with Lo
       Utils.close(kubeClient)
     }
   }
+
+  override def doTriggerSavepoint(request: TriggerSavepointRequest, flinkConfig: Configuration): SavepointResponse = {
+    flinkConfig.safeSet(DeploymentOptions.TARGET, ExecutionMode.KUBERNETES_NATIVE_APPLICATION.getName)
```

Comment:
```suggestion
    flinkConfig.safeSet(DeploymentOptions.TARGET, ExecutionMode. KUBERNETES_NATIVE_SESSION.getName)
```

Is it a bug?

Reply:
The lines is coded follow the original lines in the class.
@wolfboys could you help check it ? I'm not familiar with the k8s execution mode. As mentioned above comments, I test for the feature on all execution mode besides k8s mode.
CC @1996fanrui 
thank you.

Reply:
> Is it a bug?

Confirmed with @wolfboys offline. It's a bug. I fixed it.
PTAL.  :)
Thank you @wolfboys @1996fanrui